draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -763,29 +1013,62 @@ As a rough style guide the following convention is used:
 
 - c1, c2, ...: are constant values, which can be computed in advance.
 
-## Clearing Cofactor
+## Clearing the cofactor {#cofactor-clearing}
+
+Deterministic encodings guarantee that their outputs satisfy the elliptic
+curve equation. To obtain a point in a subgroup of order r, however, the
+cofactor must be cleared. In the description of each encoding in this section,
+the last step returns h * (x, y); this represents the cofactor clearing
+operation, which can always be performed via a scalar multiplication by h.
+Note that, for elliptic curves having a prime number of points, h=1 and no
```

Comment:
nit: maybe reference the NIST curves as an example?

Reply:
Good idea. Done.